bin/gyro

```diff
@@ -35,24 +35,12 @@ OptionParser.new do |opts|
   opts.on('-m PATH', '--model PATH', 'Specify Interface Builder .xcdatamodel file') do |xcdatamodel_path|
     options[:model] = xcdatamodel_path
   end
-  opts.on('-a PATH', '--android PATH', 'Specify Android Realm model classes dir') do |path|
-    options[:android_dir] = path
-  end
   opts.on('-t DIR', '--template DIR', 'Path to the Liquid template directrory to use, other solution is to specify default template name (like swift3, android) !') do |tempate_path|
     options[:template] = tempate_path
   end
   opts.on('-o DIR', '--output DIR', 'Path to the output directory to generate model') do |output_dir_path|
     options[:output] = output_dir_path
   end
-  opts.on('-p PATH', '--package PATH', 'Specify Android Realm model classes package name') do |path|
-    options[:package] = path
-  end
-  opts.on('-w', '--wrappers', 'To generate Java primitive wrappers for optional fields instead of primitive types') do
-    options[:wrappers] = true
-  end
-  opts.on('-x', '--annotations', 'To tag code with Android support annotations according to optional/non optional fields') do
-    options[:annotations] = true
-  end
```

Comment:
We need to document somewhere (in some Markdown file documenting each template?) how to pass those params for Android now, i.e. what are the `--param` names to use (and that are used by the Android templates provided with gyro) to achieve the similar features now